db/legacy_migrations/000_create_scheduled_calls_table.sql

```diff
@@ -1,7 +0,0 @@
-CREATE TABLE scheduled_calls_table (
```

Comment:
Given we're still using Postgres why are we removing the migrations? This would prevent us from being able to setup the database again if needed